Robinhood/Robinhood.py

```diff
@@ -136,13 +148,16 @@ def login(
 
         return False
 
+
     def logout(self):
-        """logout from Robinhood
+        """
+            Sogout from Robinhood
```

Comment:
spelling error

Reply:
Fixed - made that typo when I was using Vim